src/providers/interpretation_providers/word2vecInterpretationProvider.ts

```diff
@@ -33,20 +39,19 @@ export class Word2VecInterpretationProvider implements InterpretationProvider {
 		'cocktail',
 		'wine',
 	];
-	protected static nonAlcoholicWords: ReadonlySet<string> = new Set([
-		'coffee',
-		'non-alcoholic',
-	])
+	protected static nonAlcoholicWords: ReadonlySet<string> = new Set(['coffee', 'non-alcoholic']);
 	protected isAlcoholic(text: string): boolean {
 		const words = text.toLowerCase().split(' ');
-		return (
-			!words.some((word) => Word2VecInterpretationProvider.nonAlcoholicWords.has(word))
-			&& words.some((word) =>
+		const textIsAlcoholic = (
+			!words.some((word) => Word2VecInterpretationProvider.nonAlcoholicWords.has(word)) &&
+			words.some((word) =>
 				Word2VecInterpretationProvider.alcoholicWords.some((alcoholicWord) =>
 					this.similar(word, alcoholicWord)
 				)
 			)
-		)
+		);
+		console.log(`${text} : ${textIsAlcoholic ? "alcoholic" : "not alcoholic"}`)
```

Comment:
Should this log be here?

Reply:
It provides useful debugging, and the new messages are more useful than we had before.

I used this log to verify that memoization worked correctly, because no piece of text got logged twice